src/EditorFeatures/Core.Wpf/QuickInfo/LazyToolTip.cs

```diff
@@ -58,8 +75,11 @@ private void OnToolTipOpening(object sender, ToolTipEventArgs e)
                     Debug.Assert(_element.ToolTip == this);
                     Debug.Assert(_disposableToolTip == null);
 
+                    // We don't expect _disposableToolTip to be non-null here, but we still make sure it's not leaking
+                    _disposableToolTip?.Dispose();
```

Comment:
what about just not doing work in that case?  if hte tooltip was already created, we could just immediately exit?

Reply:
⏱️ Sure that seems reasonable